opentelemetry-prometheus/tests/integration_test.rs

```diff
@@ -406,6 +406,7 @@ fn gather_and_compare(registry: prometheus::Registry, expected: String, name: &'
     let metric_families = registry.gather();
     encoder.encode(&metric_families, &mut output).unwrap();
 
+    let expected = get_platform_specific_string(expected);
```

Comment:
nit - should we move this normalization to immediately after the file reading? Ideally, gather_and_compare should be agnostic of platform-specific conversion.

Reply:
Since the method is gathering data from `prometheus::Registry` in its definition, we would still need a conversion for the gathered data.